README.md

```diff
@@ -51,15 +51,19 @@ $ cd uptane
 $ pip install -r dev-requirements.txt
 ```
 
-Note that the demonstration now operates using ASN.1 / DER format and encoding for metadata files by default. The TUF branch in use has been switched accordingly (so please run the command above again if you have an existing installation). This can be switched back to JSON (which is human readable) by changing the tuf.conf.METADATA_FORMAT option in uptane/\_\_init\_\_.py.
+#### Updates
+When updating Uptane code, please reinstall its dependencies, as the corresponding TUF fork may be updated: `pip install --force-reinstall -r dev-requirements.txt`
```

Comment:
It's easier on the eyes if the command were on its own separate line. I recommend starting a new line after `updated:`